moderation/commands.go

```diff
@@ -1033,6 +1047,24 @@ var ModerationCommands = []*commands.YAGCommand{
 				return fmt.Sprintf("Could not find warning with ID `%d`", warningID), nil
 			}
 
+			if config.UnwarnSendToModlog && config.ActionChannel != 0 {
+				userid, err := strconv.ParseInt(warning.UserID, 10, 64)
+				if err != nil {
+					return "Failed parsing user ID", err
+				}
+				user := bot.GetUsers(parsed.GuildData.GS.ID, userid)[0]
+
+				message := parsed.Args[1].Str()
```

Comment:
Since we are now actually using the reason, should it be parsed initially with MBaseCmdSecond on line 1018? This way we’d get the trademark (no reason specified) when not set. 

While we’re at it, should there be a config to make delete warning reason optional?

Reply:
What were your thoughts about adding config to make warning reason optional?